tensorflow/lite/micro/tools/make/targets/arc/README.md

```diff
@@ -0,0 +1,216 @@
+
+Please consider the above when choosing whether to install Windows or Linux or both versions of MWDT
+
+
+## ARC EM Software Development Platform (ARC EM SDP) 
+
+This section describes how to deploy on an [ARC EM SDP board](https://www.synopsys.com/dw/ipdir.php?ds=arc-em-software-development-platform)
+
+### Initial Setup
+
+To use the EM SDP, you need the following hardware and software:
+
+#### ARC EM SDP
+More information on the platform, including ordering information, can be found [here](https://www.synopsys.com/dw/ipdir.php?ds=arc-em-software-development-platform).
+
+#### MetaWare Development Toolkit
+See [Install the Synopsys DesignWare ARC MetaWare Development Toolkit](#install-the-synopsys-designware-arc-metaWare-development-toolkit) section for instructions on toolchain installation.
+
+#### Digilent Adept 2 System Software Package
+If you wish to use the MetaWare Debugger to debug your code, you need to also install the Digilent Adept 2 software, which includes the necessary drivers for connecting to the targets.  This is available from oficial [Digilent site](https://reference.digilentinc.com/reference/software/adept/start?redirect=1#software_downloads).  You should install the “System” component, and Runtime. Utilities and SDK are NOT required.
+
+Digilent installation is NOT required if you plan to deploy to EM SDP via the SD card instead of using the debugger.
+
+#### Make Tool
+A `'make'` tool is required for both phases of deploying Tensorflow Lite Micro applications on ARC EM SDP: 
+1. Application project generation
+2. Working with generated application (build and run)
+
+For the first phase you need an environment and make tool compatible with Tensorflow Lite for Micro build system. At the moment of this writing, this requires make >=3.82 and a *nix-like environment which supports shell and native commands for file manipulations. MWDT toolkit is not required for this phase. 
+
+For the second phase, requirements are less strict. The gmake version delivered with MetaWare Development Toolkit is sufficient. There are no shell and *nix command dependencies, so Windows can be used 
+
+
+#### Serial Terminal Emulation Application
+The Debug UART port of the EM SDP is used to print application output. The USB connection provides both the debug channel and RS232 transport. You can use any terminal emulation program (like [PuTTY](https://www.putty.org/)) to view UART output from the EM SDP. 
+
+#### microSD Card
+If you want to self-boot your application (start it independently from a debugger connection), you also need a microSD card with a minimum size of 512 MB  and a way to write to the card from your development host
+
+### Connect the Board
+
+1.	Make sure Boot switches of the board (S3) are configured in the next way:
+<center>
+
+| Switch #  |   Switch position  |
+| :-------: | :----------------: | 
+|     1     |       Low (0)      | 
+|     2     |       Low (0)      | 
+|     3     |       Low (0)      | 
```

Comment:
is this correct?

Reply:
Nope, 3rd switch must be in HIGH (1) state. Fixed it